controller_manager/src/ros2_control_node.cpp

```diff
@@ -41,16 +39,12 @@ int main(int argc, char ** argv)
   // converted back to a timer.
   std::thread cm_thread([cm]() {
     // load controller_manager update time parameter
-    int update_rate = DEFAULT_UPDATE_RATE;
-    if (!cm->get_parameter("update_rate", update_rate))
-    {
-      RCLCPP_WARN(cm->get_logger(), "'update_rate' parameter not set, using default value.");
-    }
-    RCLCPP_INFO(cm->get_logger(), "update rate is %d Hz", update_rate);
+    cm->configure();
+    RCLCPP_INFO(cm->get_logger(), "update rate is %d Hz", cm->get_update_rate());
 
     std::chrono::system_clock::time_point timepoint_start = std::chrono::system_clock::now();
     std::chrono::system_clock::time_point begin = timepoint_start;
-    std::this_thread::sleep_for(std::chrono::nanoseconds(1000000000 / update_rate));
+    std::this_thread::sleep_for(std::chrono::nanoseconds(1000000000 / cm->get_update_rate()));
```

Comment:
```suggestion
    // Use nanoseconds to avoid chrono's rounding
    std::this_thread::sleep_for(std::chrono::nanoseconds(1000000000 / cm->get_update_rate()));
```